platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/internal/EventImpl.java

```diff
@@ -132,7 +132,7 @@ public Hash getBaseHash() {
     /**
      * @return array of transactions inside this event instance
      */
-    public ConsensusTransactionImpl[] getTransactions() {
```

Comment:
we should get rid of the arrays if we can

Reply:
done